pkg/kubectl/cmd/create_deamonset.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/kubernetes/pkg/kubectl"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+)
+
+var (
+	daemonsetLong = templates.LongDesc(i18n.T(`
+	Create a daemonset with the specified name.
+
+	A DaemonSet ensures that all (or some) nodes run a copy of a pod.  As nodes are added to the
```

Comment:
I doubt it's necessary to repeat docs here. Link to docs is fine, but I don't recall any create command that has it, so I'd drop it as well. The long description is usually one full sentence what the command does and that's it. The examples is where you focus on usage. All in all if you reach out for `--help` you'll get sufficient information.